### DISCUSSION TITLE
Where is PPM_decode defined?

### DISCUSSION THREAD

**Discussion by daveduval** [pascallanger/DIY-Multiprotocol-TX-Module#928] [General]
I'm a beginner programmer trying to understand how the project works.

Where is the PPM_decode function defined? Is there a general technique to answering this kind of question by searching through the files of the project?

**Comment by pascallanger** [comment]
The ppm deciding is done in multiprotocol.ino using interrupts.

**Reply by daveduval** [reply]
Thank you!
